### ISSUE TITLE
[BUG] Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value for `updatePopupAction`

### ISSUE THREAD

**Issue by mattisssa** [Mijick/Popups#167]
## Prerequisites
- [x] I checked the [documentation](https://github.com/Mijick/Popups/wiki) and found no answer
- [x] I checked to make sure that this issue has not already been filed

## Steps to Reproduce
Please provide detailed steps for reproducing the issue.

I tried to implement the exact same structure as shown in the documentation, but the app crashed.

## Code Sample
If you can, please include a code sample that we can use to debug the bug.

From framework's ViewModel:
```
func updatePopupHeight(_ heightCandidate: CGFloat, _ popup: AnyPopup) async {
        guard activePopupProperties.gestureTranslation == 0 else { return }

        let newHeight = await calculatePopupHeight(heightCandidate, popup)
        if newHeight != popup.height {
            await updatePopupAction(popup.updatedHeight(newHeight))
        }
    }
```

| Name | Version |
| ------| ---------|
| Xcode | Version 16.2 (16C5032a) |
| Operating System | iOS 18.1 |
| Device | iPhone 16 Pro |

**Comment by FulcrumOne** [comment]
Hey @mattisssa,

I decided to protect the library against such situations and implemented support for this in patch 4.0.1.

Have a nice day,
Tomasz

**Comment by mattisssa** [comment]
Awesome, thanks!
